nfsstat/tests/compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  nfs-server:
+    image: gists/nfs-server
+    container_name: nfs-server
+    privileged: true
+    ports:
+      - 2049:2049
+    volumes:
+      - ./nfs-share:/nfs-share
+
+  nfs-client:
+    image: ubuntu:18.04
```

Comment:
Is there a lighter equivalent image we can use instead ? An Alpine (or debian) image for example ?